### PR TITLE
Added phantomsObeyHostileMobCap rule

### LINKED ISSUE #2: Phantom's obey the hostile mobcap
Proposal:
- Carpet rule which makes phantoms obey the hostile mobcap.

Motivation:
- I hate phantoms

Details:
- Rule was implemented in [lunaar carpet addons](https://github.com/Lunaar-SMP/lunaar-carpet-addons) but hasn't been updated since 1.18. Worth potentially re-using this code.

### PR DESCRIPTION
**Pull request by Gilly7CE**
Phantoms will no longer spawn around a player if their hostile mobcap is full.

Closes #2

### CODE REVIEW

**Comment by Gilly7CE** [comment]
Just realised the ChunkManagerHelper class is probably not needed now. That was required for the lunaar port.

**Comment by Gilly7CE** [comment]
> Just realised the ChunkManagerHelper class is probably not needed now. That was required for the lunaar port.

Nevermind, only some method usages are no longer needed.

**Comment by Gilly7CE** [comment]
> > Just realised the ChunkManagerHelper class is probably not needed now. That was required for the lunaar port.
> 
> Nevermind, only some method usages are no longer needed.

Removed.

File: README.md
Comment (Serzbiedry) [inline README.md:1]: Isn't this allready on main branch?

**Review by Serzbiedry — APPROVED** [review]

File: README.md
Comment (Gilly7CE) [reply README.md:1]: Yes, it has just been moved into alphabetical order.

Comment (Gilly7CE) [reply README.md:1]: And the category was changed from `SURVIVAL` to `FEATURE`, as this applies to creative mode too.

**Pull request merged**